lib/screens/app/wallet/dashboard/wallet_header.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:carousel_slider/carousel_controller.dart';
+import 'package:carousel_slider/carousel_options.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:dots_indicator/dots_indicator.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+import '../../../../widgets/dashboard_widgets/currency_info_card.dart';
+
+class WalletHeader extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return WalletHeaderState();
+  }
+}
+
+class WalletHeaderState extends State<WalletHeader> {
+  final CarouselController _controller = CarouselController();
+  int _selectedIndex = 0;
+
+  ScrollController dd = ScrollController();
+
+  void onPageChange(int index, CarouselPageChangedReason changeReason) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        SingleChildScrollView(
+          child: CarouselSlider(
+            carouselController: _controller,
+            items: [
+              const Padding(
+                padding: EdgeInsets.only(right: 10),
+                child: CurrencyInfoCard(
+                  cardWidth: 380,
```

Comment:
Do we need the card width? Can we create this using percentages ?

Reply:
cardView is not need it since viewportFraction will give the card width, however for some reason the card background disappear and it came back after I added the width. I took the widht out and is working now with out it